src/components/Title.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { render, RenderResult, screen } from '@testing-library/react';
+
+import Title, { TitleProps } from './Title';
+
+function renderTitle(props: Partial<TitleProps> = {}): RenderResult {
+  return render(<Title testId="title">{props.children}</Title>);
+}
+
+describe(`Title`, () => {
+  it(`renders a Title component`, () => {
+    renderTitle();
+
+    expect(screen.getByTestId(`title`)).toBeInTheDocument();
+  });
+
+  it(`renders text children`, () => {
+    renderTitle({ children: `Test Title` });
+
+    expect(screen.getByTestId(`title`)).toHaveTextContent(`Test Title`);
```

Comment:
I usually don't like to use `data-testid`, unless there's no option. 
You could do something like this here:
```
expect(getByText('Test Title')).toBeInTheDocument()
```

Reply:
Done